corehq/apps/app_manager/suite_xml/post_process/instances.py

```diff
@@ -1,3 +1,39 @@
+"""
+Notes on instances
```

Comment:
adding this here for now - I'm not quite sure if there's a better place for it.

Reply:
This seems good. For the suite generation session we did recently, I was originally going to put the content as autodoc in these modules (and I still should probably do that, just need to clean up the content a bit).

Reply:
putting it [here](https://github.com/dimagi/commcare-hq/tree/master/docs/apps) also seems like a fine option